.github/workflows/ci.yml

```diff
@@ -68,18 +68,6 @@ jobs:
           pip install virtualenv
           # This will also trigger "make dist" that creates the Python packages
           make aws-lambda-layer
-
-          echo "Saving SDK_VERSION for later"
-          export SDK_VERSION=$(grep "VERSION = " sentry_sdk/consts.py | cut -f3 -d' ' | tr -d '"')
-          echo "SDK_VERSION=$SDK_VERSION"
-          echo "SDK_VERSION=$SDK_VERSION" >> $GITHUB_ENV
-      - name: Upload Python AWS Lambda Layer
-        uses: getsentry/action-build-aws-lambda-extension@v1
```

Comment:
This gh action is also used in Javascript to create the excentions including relay: https://github.com/getsentry/sentry-javascript/blob/b9fb55f6c916ada89ec3c871078cc45661380f42/.github/workflows/build.yml#L285-L291

The idea behind this was to have one GH action to create the extension and include it in all the layers. 

So we can aim for two things now:
a) We remove this action in JS and remove also the actions repo as a whole. So going forward each language has to create the lambda layer on there own.
b) Keep the action as the single thing that takes a directory and makes a lambda layer out of it, so the different languages do not need to know how this is done. (having the layer zipping logic in one place only)

As without the lambda extension the action is only doing the zipping, i am fine with either option.

Reply:
see TODO list in linked issue
https://github.com/getsentry/team-webplatform-meta/issues/58